projects/plugins/jetpack/modules/shortcodes/crowdsignal.php

```diff
@@ -240,22 +240,28 @@ public function crowdsignal_shortcode( $atts ) {
 			 * Rating embed.
 			 */
 			if ( (int) $attributes['rating'] > 0 && ! $no_script ) {
+				$post_id = $post instanceof WP_Post ? $post->ID : get_the_ID();
```

Comment:
Since `get_the_ID()` can return false, should we add an additional default to avoid notices?